.github/workflows/tests.yaml

```diff
@@ -21,8 +21,7 @@ jobs:
         # TODO: include highest & lowest
         dependencies: [ locked ]
         coverage-driver: [ pcov, xdebug ]
-        # TODO: include Symfony6 later
-        symfony-version: [ '5.4.*' ]
+        symfony-version: [ '5.4.*', '6.*.*' ]
```

Comment:
not sure I understand how it works now.

All the builds have now `locked` value for `dependencies` key. Each GH job runs `composer install`, ignoring (as I see) SF 5.4.* or 6..* - jobs are just use the versions from the lock file.

- should we add `highest` and `lowest` on line 21?
- should we vary SF versions **only** when `dependencies` is not `locked`?

Reply:
Resolved in #1835